go/vt/vtgateproxy/vtgateproxy.go

```diff
@@ -202,8 +204,10 @@ func (proxy *VTGateProxy) StreamExecute(ctx context.Context, session *vtgateconn
 
 func Init() {
 	log.V(100).Infof("Registering GRPC dial options")
+
 	grpcclient.RegisterGRPCDialOptions(func(opts []grpc.DialOption) ([]grpc.DialOption, error) {
-		return append(opts, grpc.WithDefaultServiceConfig(`{"loadBalancingConfig": [{"round_robin":{}}]}`)), nil
+		log.Infof("registering %s load balancer", *balancerType)
+		return append(opts, grpc.WithDefaultServiceConfig(fmt.Sprintf(`{"loadBalancingConfig": [{"%s":{}}]}`, *balancerType))), nil
```

Comment:
we should sanitize the input here - I'm not sure what the behaviour of gRPC is if you ask it for a balancer it doesn't know about.

Reply:
good point. done.